### PR TITLE
Update __init__.py

### PR DESCRIPTION
**Pull request by akurach**
add support for unicode symbols in vm names.
fix for '''UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 5: ordinal not in range(128)'''

### What does this PR do?
add support for unicode symbols in vm names.
fix for '''UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 5: ordinal not in range(128)'''

### Previous Behavior
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2013' in position 5: ordinal not in range(128)

### New Behavior
works!

### Tests written?

No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/latest/topics/development/contributing.html) for best practices.

### CODE REVIEW

File: salt/cloud/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 '''
 
 # Import python libs
-from __future__ import absolute_import, print_function, generators
+from __future__ import absolute_import, print_function, generators,unicode_literals
```
Comment (gtmanfred) [inline salt/cloud/__init__.py]: you need a space after the comma and then pylint will pass

**Review by gtmanfred — CHANGES_REQUESTED** [review]

**Comment by damon-atkins** [comment]
Keep in mine you can not general mix PY2 Strings and Unicode.   May need to test all external input is Unicode and if it is not convert it to Unicode.
 
In theory develop is moving to Unicode by default ( @terminalmage  )

modules win_pkg.py and reg.py have support Unicode since 2016.11

**Comment by akurach** [comment]
@damon-atkins hi, i got that error while do salt-cloud -a show_instance vmname. 
seems like one(ore more) of our customers create vm with non-eng symbol. And when salt tries to find $vmname it falls down.

**Comment by akurach** [comment]
@gtmanfred done)

**Review by gtmanfred — APPROVED** [review]

**Review by rallytime — APPROVED** [review]
